### ISSUE TITLE
Add further examples

### ISSUE THREAD

**Issue by bkmgit** [tensorly/tensorly.github.io#2]
It may be helpful to add/link further examples from
- https://github.com/JeanKossaifi/caltech-tutorial 
- https://github.com/JeanKossaifi/tensorly-notebooks
- https://github.com/JeanKossaifi/caltech-tutorial-19
- https://github.com/tensorly/Proceedings_IEEE_companion_notebooks

Choosing one set of examples for the documentation would make learning and maintenance upgrades easier

**Comment by JeanKossaifi** [comment]
I agree, adding more examples is on my todo list! Feel free to have a crack at it though! :)
